File: generated_code_examples/c/classification/decision_tree.c
#include <string.h>
void score(double * input, double * output) {
    double var0[3];
    if (input[2] <= 2.449999988079071) {
        memcpy(var0, (double[]){1.0, 0.0, 0.0}, 3 * sizeof(double));
    } else {
        if (input[3] <= 1.75) {
            if (input[2] <= 4.950000047683716) {
                if (input[3] <= 1.6500000357627869) {
                    memcpy(var0, (double[]){0.0, 1.0, 0.0}, 3 * sizeof(double));
                } else {
                    memcpy(var0, (double[]){0.0, 0.0, 1.0}, 3 * sizeof(double));
                }
            } else {
                memcpy(var0, (double[]){0.0, 0.3333333333333333, 0.6666666666666666}, 3 * sizeof(double));
            }
        } else {
            memcpy(var0, (double[]){0.0, 0.021739130434782608, 0.9782608695652174}, 3 * sizeof(double));
        }
    }
    memcpy(output, var0, 3 * sizeof(double));
}

File: generated_code_examples/lua/classification/decision_tree.lua
function score(input)
    local ml = {}
    if input[3] <= 2.449999988079071 then
        ml.var0 = {1.0, 0.0, 0.0}
    else
        if input[4] <= 1.75 then
            if input[3] <= 4.950000047683716 then
                if input[4] <= 1.6500000357627869 then
                    ml.var0 = {0.0, 1.0, 0.0}
                else
                    ml.var0 = {0.0, 0.0, 1.0}
                end
            else
                ml.var0 = {0.0, 0.3333333333333333, 0.6666666666666666}
            end
        else
            ml.var0 = {0.0, 0.021739130434782608, 0.9782608695652174}
        end
    end
    return ml.var0
end
